packages/core/src/App/Containers/ResetEmailModal/confirm-email-modal.jsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { SentEmailModal } from '@deriv/account';
+import { WS } from 'Services';
+import PropTypes from 'prop-types';
+import { connect } from 'Stores/connect';
+
+export const ConfirmEmailModal = ({ onClose, is_open, prevEmail, newEmail, verification_code, pullErrorMessage }) => {
```

Comment:
since prevEmail and newEmail are variables, can we change them to prev_email and  new_email ?